deploy/helm/elastic-agent/examples/fleet-managed-ksm-sharding/README.md

```diff
@@ -22,7 +22,7 @@ In this example we will perform two Helm chart installations, one installing ela
       --set kubernetes.state.agentAsSidecar.enabled=true \
       -n kube-system
     ```
-3. In the associated policy from the previous steps install the Kubernetes integration and **enable only** the "Collect Kubernetes metrics from kube-state-metrics".
+3. In the associated policy from the previous steps install the Kubernetes integration and **enable only** the "Collect Kubernetes metrics from kube-state-metrics". Update the `Host` destination in all data streams to `localhost:8080` instead of the default `kube-state-metrics:8080`.
```

Comment:
```suggestion
3. In the associated policy from the previous steps install the Kubernetes integration and **enable only** the "Collect Kubernetes metrics from kube-state-metrics" input. For all associated data streams, update the `Host` destination to `localhost:8080` (instead of the default `kube-state-metrics:8080`), and disable the **Leader Election** option.
```